global.css

```diff
@@ -210,11 +211,14 @@ select[disabled] + .select-arrow {
 }
 
 :focus,
-.CodeMirror-focused {
+.CodeMirror-focused,
+[data-focused-via-click] input[type="text"]:focus,
+[data-focused-via-click] input[type="number"]:focus {
```

Comment:
just in case, `search` isn't needed here?

Reply:
These are for usercss config, and option labels, so I don't think so. Couldn't hurt though.